src/constants.ts

```diff
@@ -10,6 +10,7 @@ export const PARAM_REDUX_MERGE_STATE = 'PARAM_REDUX_MERGE_STATE'
 export const ARG_REDUX_PATH = 'ARG_REDUX_PATH'
 
 export const ACTIONS_TYPES = {
+  RESET_REDUX_TYPE: '@@RESET_REDUX',
```

Comment:
Change this to '@@WITH_REDUX_RESET' to match the format of the other actions. I wish I had done the same for SET_STATE_AT_PATH_TYPE